### PR TITLE
fix: fix sparse float vector column warmup

### PR DESCRIPTION
**Pull request by sunby**
https://github.com/milvus-io/milvus/issues/35438

### CODE REVIEW

File: internal/core/src/mmap/Column.h [hunk 1]
```diff
@@ -102,7 +102,6 @@ class ColumnBase {
                storage::MmapChunkDescriptorPtr descriptor)
```
Comment (zhengbuqian) [inline internal/core/src/mmap/Column.h:102]: update comments for this ctor:

```cpp
// use mmap manager ctor, used in growing segment fixed data type and sparse float vector type.
// type_size_ is not initialized/used for sparse float vector type.
```

File: internal/core/src/mmap/Column.h [hunk 2]
```diff
@@ -502,6 +504,18 @@ class SparseFloatColumn : public ColumnBase {
...
+    void
+    AppendBatchMmap(const FieldDataPtr data) {
+        AssertInfo(data->Size() + size_ <= cap_size_,
+                   "append batch mmap exceed");
+        for (size_t i = 0; i < data->get_num_rows(); ++i) {
+            auto vec = static_cast<const knowhere::sparse::SparseRow<float>*>(
+                data->RawValue(i));
+            memcpy(data_ + size_, vec->data(), vec->data_byte_size());
+            size_ += vec->data_byte_size();
+        }
```
Comment (zhengbuqian) [inline internal/core/src/mmap/Column.h:516]: update dim_

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**